.github/workflows/glean.yaml

```diff
@@ -0,0 +1,25 @@
+name: GLEAN probe-scraper
```

Comment:
Any particular reason we're styling Glean in all-caps?

Reply:
for some reason i incorrectly thought that's how Glean was supposed to be styled. will fix.